templates/bundles/PrestaShopBundle/Admin/Product/CatalogPage/Lists/products_table.html.twig

```diff
@@ -85,7 +85,22 @@
       {% block product_catalog_form_table_filters %}
         {% set filters_disabled = activate_drag_and_drop %}
         <tr class="column-filters">
-          <th colspan="2">
+          <th class="text-center" style="vertical-align: middle;">
+            {% block product_catalog_filter_select_all %}
+              <div class="md-checkbox md-checkbox-inline">
+                <label>
+                  <input
+                    type="checkbox"
+                    id="bulk_action_select_all"
+                    onclick="$('#product_catalog_list').find('table td.checkbox-column input:checkbox').prop('checked', $(this).prop('checked')); updateBulkMenu();"
```

Comment:
I'm really not a big fan of adding onclick directly in the dom when you can do something simple in js files :thinking:

Reply:
@PierreRambaud I only moved the code already present. It will be redone when migrating this page to grid. ;-)

Reply:
haha didn't notice that, I'm sorry :sweat_smile: